library/maven

```diff
@@ -1,58 +1,119 @@
 Maintainers: Carlos Sanchez <carlos@apache.org> (@carlossg)
 GitRepo: https://github.com/carlossg/docker-maven.git
 
+Tags: 3.6.3-jdk-11, 3.6-jdk-11, 3-jdk-11
+Architectures: amd64, arm64v8
+GitCommit: 26ba49149787c85b9c51222b47c00879b2a0afde
+Directory: jdk-11
```

Comment:
These directory names need to change too, right? I think they're `openjdk-*` now?  (Sorry I didn't notice before -- with all GitHub's issues this morning I thought it was GitHub failing and have been restarting the job over and over :see_no_evil:)

Reply:
they are symlinks but I see now they don't work, I'll change